functional-tests/tests/tests.js

```diff
@@ -1,8 +1,118 @@
 module.exports = {
-	'Nightwatch.js Test': function (browser) {
-		browser
-		  .url(process.env.BAG_CUPID_URL)
-		  .assert.title('Bag Cupid')
-		  .end();
-	}
+	'Check for page title and Banner': function (browser) {
+		browser.url(process.env.BAG_CUPID_URL);
+		browser.assert.title('Bag Cupid');
+		browser.pause(6000);
```

Comment:
We can get this merged as-is with these `pause`s everywhere, but as a next step we should try to remove them by using nightwatch's wait until syntax.  At the moment, some of these tests take almost 20 seconds when they really should run in under a second (assuming fast network).  I have some changes I want to do in the near future to make running the tests locally much faster as well, but they won't help if we are using `pause` instead of the `wait until` stuff.

Lets leave that to a follow-up Pull Request though, for now just address the other couple minor comments.